src/app-router.js

```diff
@@ -27,8 +27,8 @@ import { triggerMethod }       from './trigger-method';
 
 var AppRouter = Backbone.Router.extend({
 
-  constructor: function(options) {
-    this.options = options || {};
+  constructor: function(options = {}) {
```

Comment:
hmm, I think some people use `this.options` so it'd be nice to keep the default behavior